typescript-primitives-and-variables/typescript-primitives-and-variables-quiz-notes.md

```diff
@@ -0,0 +1,60 @@
+# typescript-primitives-and-variables-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is the purpose of variables?
+
+- How do you **declare** a variable?
+
+- How do you initialize (**assign a value to**) a variable?
+
+- What characters are allowed in variable names?
+
+- What does it mean to say that variable names are "case sensitive"?
+
```

Comment:
Missing a few quiz questions here! Make sure to include answers for all of them. I'm not necessarily looking for the "correct" answer, but I would like you to submit something, because it helps give me an awareness of what you got out of this!